ui/webui/src/components/review/ReviewConfiguration.scss

```diff
@@ -1,27 +1 @@
-/* Style a PF expandable section as a header not a link  */
-.review-expandable-section .pf-c-expandable-section__toggle-text {
-	color: var(--pf-global--Color--100);
-}
-
-/* Override description bold look */
-.description-list-term {
-	font-weight: var(--pf-global--FontWeight--normal);
-}
-
-.description-list-description {
-	font-size: var(--pf-c-description-list__term--FontSize);
-}
-
-.storage-devices-configuration-title {
-	margin-top: var(--pf-global--spacer--md);
-	margin-bottom: var(--pf-global--spacer--sm);
-}
-
-.storage-devices-divider {
-	--pf-c-divider--BorderColor--base: var(--pf-global--Color--100);
-	--pf-c-divider--BackgroundColor: var(--pf-global--Color--100);
-}
-
-.review-disk-label {
-	font-weight: var(--pf-global--FontWeight--bold);
-}
+@use "../../node_modules/@patternfly/patternfly/utilities/Text/text.css";
```

Comment:
In case you repost you can:
use "@patternfly/patternfly/utilities/Text/text.css";